CONTRIBUTING.md

```diff
@@ -1,10 +1,38 @@
 
-Thank you for considering a contribution to Subiquity.  Here's what we like to
-see:
+Thank you for considering a contribution to Subiquity.  Here's some things to
+note:
 
-* Contributors will need to have signed the
-  [CLA](https://ubuntu.com/legal/contributors/agreement).
-* Format, lint, unit, integration, and API tests should be passing.
+## Code of Conduct
+
+This project is subject to the [Ubuntu Code of Conduct](https://ubuntu.com/community/code-of-conduct)
+to foster an open and welcoming place to contribute. By participating in the
+project (in the form of code contributions, issues, comments, and other
+activities), you agree to abide by its terms.
+
+## Contributor License Agreement
+
+This project is subject to the
+[Canonical contributor license agreement](https://ubuntu.com/legal/contributors),
+please make sure you have [signed it](https://ubuntu.com/legal/contributors/agreement)
+before (or shortly after) submitting your first pull request.
+
+A github workflow will verify that your GitHub username or email address is
+associated with a contributor license agreement signature, but it may take
+some time after your initial signature for the check to see it. If you're
+part of @canonical, you will also need to make sure your @canonical.com
+email address is associated with your GitHub account.
+
+## Bugs
+Bugs are tracked on [Launchpad](https://bugs.launchpad.net/subiquity). It is
+recommended you use `ubuntu-bug` (or similar) to let apport collect relevant
+logs which are helpful for the debug process, instead of filing one directly.
+
+## Pull Requests
+Changes to this project should be proposed as pull requests on GitHub at:
+[https://github.com/canonical/subiquity/](https://github.com/canonical/subiquity/)
+
+
+ Format, lint, unit, integration, and API tests should be passing.
   * format - either run `make format` or `pre-commit install`.
     See [pre-commit](https://pre-commit.com/#install) for more details on that
```

Comment:
Not a problem for this PR obviously, but while we're here, can we point people to the archive version of pre-commit instead of the non-archive one?  `apt install pre-commit` works well, would you update that part?

Reply:
Sure thing.